src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -695,6 +695,19 @@ public void deleteFeedbackResponsesForCourses(List<String> courseIds) {
         return (List<FeedbackResponse>) q.execute(courseId);
     }
     
+    public boolean hasFeedbackResponseEntitiesForCourse(String courseId) {
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, courseId);
+        
+        Query q = getPm().newQuery(FeedbackResponse.class);
```

Comment:
Perhaps this portion (line 701 -707) can be extracted as `getFeedbackResponseEntitiesForCourseWithinRange`, so that the method could be reused in the future.